### PR TITLE
Code for div should be in footer not in <head>

### PR DESCRIPTION
**Pull request by iranzo**
Fix operation moving code to footer

### CODE REVIEW

**Comment by iranzo** [comment]
It should also be applied to 'master' as the bug is there

**Comment by iranzo** [comment]
I did validate on my site, and seemed to work fine

File: templates/_includes/footer.html [hunk 1]
```diff
@@ -12,4 +12,9 @@
         <li class="elegant-power">Powered by <a href="http://getpelican.com/" title="Pelican Home Page">Pelican</a>. Theme: <a href="https://github.com/Pelican-Elegant/elegant/" title="Theme Elegant Home Page">Elegant</a></li>
     </ul>
 </div>
+{% if AMAZON_ONELINK %}
+<div id="amzn-assoc-ad-{{ AMAZON_ONELINK }}"></div>
```
[the remark this answers is not in the document]
Comment (iranzo) [reply templates/_includes/footer.html]: I can make test, it is said to use it, but then it's open/closed unless filled with js dynamically, but even on my site, it's empty

**Comment by iranzo** [comment]
Without the 'div' the browser complains with:

image

File: templates/_includes/footer.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (iranzo) [reply templates/_includes/footer.html]: It didn't worked as I put in the capture, seems that it's needed for script to load asynchronously

[the remark this answers is not in the document]
Comment (talha131) [reply templates/_includes/footer.html]: Thank you

**Comment by iranzo** [comment]
Done @talha131

**Comment by talha131** [comment]
@iranzo one change I requested is still open.

**Comment by iranzo** [comment]
Changelog is updated, and the script is at footer as amazon doc it says that it should be before </body>.

As it loads asynchronously it should not affect page load (in fact, amazon docs, say that if you use 'sync' it will cause slower page load).

Is this what was missed @talha131 ?

**Comment by talha131** [comment]
> Is this what was missed @talha131 ?

My bad. After requesting change, I missed to click on "Finish review" so it was visible to me but not to you.

The request for change should be visible now.

**Comment by iranzo** [comment]
Done!

**Pull request merged**